doc/tutorials/vuart_configuration.rst

```diff
@@ -214,3 +214,131 @@ started, as shown in the diagram below:
    the hypervisor is not sufficient. Currently, we recommend that you use
    the configuration in the figure 3 data flow. This may be refined in the
    future.
+
+Use PCI-vUART
+#############
+
+PCI Interface of ACRN vUART
+===========================
+
+When user set `vuart[0] and vuart[1] <https://projectacrn.github.io/latest/tutorials/vuart_configuration.html#introduction>`_,
```

Comment:
Please use a reference to a label when cross-referencing documents/paragraphs

Reply:
OK, will use reference label